### DISCUSSION TITLE
Where to put helpful resources for learning the Charm stack?

### DISCUSSION THREAD

**Discussion by bashbunni** [charm-community/.github#5] [Q&A]
https://github.com/charm-community/charm-in-the-wild
is a repo for housing projects built with Charm tools to serve as examples for others and help connect the community for collaboration.

That being said, should there be a section of this repo dedicated to helpful videos, blog posts, etc. from the community? If not, where else should that information live?

(mentioned in the Discord)

**Comment by wissam** [comment]
In my opinion, depends on the volume we get. Now we added the first blog post in a tutorial category below everything else. If we start having a huge volume, maybe we can migrate them to another repo for tutorials only . If we want to be fancy, we can have it on a page along the line of https://community-tutorials.charm.sh  or something similar.

**Reply by Delta456** [reply]
I agree with @wissam here. We should start with the accumulation of tutorials if it gets larger with time. We can move to a separate repo and even have our blogs/tutorials by the community members.
